Resources/public/js/views/ez-languageselectionboxview.js

```diff
@@ -0,0 +1,278 @@
+                'tap': '_cancel',
+            },
+            '.ez-languageselectionbox-confirm': {
+                'tap': '_confirm',
+            },
+        },
+
+        /**
+         * Tap event handler on the close links. It fires the `cancel` event.
+         *
+         * @method _cancel
+         * @protected
+         * @param {EventFacade} e
+         */
+        _cancel: function (e) {
+            e.preventDefault();
+            this.fire(CANCEL);
+        },
+
+        /**
+         * Tap event handler on the confirm button. It fires the `confirm` event.
+         *
+         * @method _confirm
+         * @protected
+         * @param {EventFacade} e
+         */
+        _confirm: function (e) {
+            e.preventDefault();
+            this.fire(CONFIRM);
+        },
+
+        initializer: function () {
+            this.on(['confirmHandlerChange', 'cancelHandlerChange'], function (e) {
+                this._syncEventHandler(e.attrName.replace(/Handler$/, ''), e.prevVal, e.newVal);
+            });
+            this._syncEventHandler('confirm');
+            this._syncEventHandler('cancel');
+            this.after('activeChange', function () {
+                if ( this.get('active') ) {
+                    this._uiUpdateTitle();
+                    this._uiUpdateExistingTranslations();
+                    this._uiUpdateNewTranslations();
+                }
+            });
+            this._publishEvents();
+        },
+
+        /**
+         * Updates the title in the rendered selectionLanguageBox
+         * 
+         * @method _uiUpdateTitle
+         * @protected
+         */
+        _uiUpdateTitle: function () {
+            this.get('container')
+                .one('.ez-languageselectionbox-title').setContent(this.get('title'));
+        },
+
+        _uiUpdateExistingTranslations: function () {
+            var translationsContainer = this.get('container').one('.ez-languageselectionbox-existingtranslations'),
+                that = this;
+
+            translationsContainer.setContent('');
+
+            Y.Array.each(this.get('existingTranslations'), function (languageCode) {
+                translationsContainer.append(that._getSingleTranslation(languageCode, 'ez-existingtranslation'));
+            });
+        },
+
+        _uiUpdateNewTranslations: function () {
+            var translationsContainer = this.get('container').one('.ez-languageselectionbox-newtranslations'),
+                that = this;
+
+            translationsContainer.setContent('');
+
+            Y.Array.each(this.get('newTranslations'), function (languageCode) {
+                translationsContainer.append(that._getSingleTranslation(languageCode, 'ez-newtranslation'));
+            });
+        },
+
+        _getSingleTranslation: function (languageCode, className) {
+            var node = Y.Node.create(this.TRANSLATION_TPL),
+                flagCode = languageCode.substring(4, 6).toLowerCase();
```

Comment:
by generating [the languages list in the template](https://github.com/ezsystems/PlatformUIBundle/pull/280/files#r34758029) you'll be able to use the Handlebars helper instead of duplicating its code here.